src/CtrlS/RoundState.java

```diff
@@ -77,4 +92,8 @@ public long getRoundTime() {
     public int getRoundCurrency() {
         return roundCurrency;
     }
+
+    public void levelBonusIN(){
+        levelBonus2 += shipStatus.getCoinIn();
```

Comment:
Please check it can be builded.
There are still some error in file. such like this line.